tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -22,6 +23,62 @@ static void Main(string[] args)
         {
             var options = JitBenchHarnessOptions.Parse(args);
 
+            SetupStatics(options);
+
+            using (var h = new XunitPerformanceHarness(args))
+            {
+                ProcessStartInfo startInfo = options.UseExistingSetup ? UseExistingSetup() : CreateNewSetup();
+
+                string scenarioName = "MusicStore";
+
+                if (options.EnableTiering)
+                {
+                    startInfo.Environment.Add("COMPlus_EXPERIMENTAL_TieredCompilation", "1");
+                    scenarioName += " Tiering";
+                }
+
+                if (options.Minopts)
+                {
+                    startInfo.Environment.Add("COMPlus_JITMinOpts", "1");
+                    scenarioName += " Minopts";
+                }
+
+                if (options.DisableR2R)
+                {
+                    startInfo.Environment.Add("COMPlus_ReadyToRun", "0");
```

Comment:
You should probably add a comment saying that this disables readyToRun native images. 

And below saying that NoNgen just disables fragile native images. 

And in practice you (always) need both because System.Private.Corelib is a fragile native image and all the others are ReadyToRun native images.

Reply:
The options (at least when invoking the harness from the command line) are documented:
```
D:\repos\coreclr2\bin\sandbox>corerun JitBench.exe --help
JitBenchHarness

  --use-existing-setup     Use existing setup.
  --tiering                Enable tiered jit.
  --minopts                Force jit to use minopt codegen.
  --disable-r2r            Disable loading of R2R images.
  --disable-ngen           Disable loading of ngen images.
  --iterations             Number of iterations to run.
  -o                       Specifies the intermediate output directory name.
  --target-architecture    Required. JitBench target architecture (It must
                           match the built product that was copied into
                           sandbox).
  --help                   Display this help screen.
  --version                Display version information.
```
Are you saying that just disabling ngen w/o disabling R2R is uninteresting or won't work?

Reply:
I'm saying if you have a normal optimized install you will have exactly one fragile ngen image and N different ReadyToRun native images. 

When I wanted to get a 100% JIT run I had to disable both kinds of native images.
Only disabling one kind of native image, didn't seem to me to be an interesting measurement.

Reply:
Ok, got it.

These options are for manual exploration for now so I'm going to keep them separate. If it turns out I never need to disable R2R/NGEN seperately I'll consider merging them into just one option.